src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +9,25 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new RuntimeException("User can not be null");
+        }
+
+        if (user.getLogin() == null || user.getLogin().length() < 6) {
```

Comment:
The check for the login length should be split into two separate conditions. One to check if the login is null and another one to check if the length is less than 6. This will provide clearer messages for different validation failures.